src/main/java/com/labsynch/labseer/domain/AbstractBBChemStructure.java

```diff
@@ -87,13 +87,10 @@ public void setId(Long id) {
 
     private String registrationComment;
 
-    @Transient
```

Comment:
Making these not transient makes it so hibernate stores the property in the table which we now use when standardizing to make it so we don't have to call cmpdStructureService a second time when doing the actual standardization after dry run.